src/stream.js

```diff
@@ -3,9 +3,19 @@ const config = require('./config');
 const SCAudio = require('../vendor/playback/playback').SCAudio;
 const StreamUrlRetriever = require('../vendor/playback/playback').SCAudioPublicApiStreamURLRetriever.StreamUrlRetriever;
 const MaestroHTML5Player = require('../vendor/playback/playback').MaestroHTML5Player.HTML5Player;
-// const MaestroHLSMSEPlayer = require('../vendor/playback/playback').MaestroHLSMSEPlayer.HLSMSEPlayer;
+const MaestroHLSMSEPlayer = require('../vendor/playback/playback').MaestroHLSMSEPlayer.HLSMSEPlayer;
 const stringLoader = require('../vendor/playback/playback').MaestroLoaders.stringLoader;
 
+function registerPlay() {
+  let registerEndpoint = `${baseURL}/tracks/${encodeURIComponent(track.id)}/plays?client_id=${encodeURIComponent(clientId)}`;
```

Comment:
It was, but it doesn't work because it is a post request, and the api module wants to add the oauth token and also other things as form data, instead of query params.